lib/flak/assist.go

```diff
@@ -0,0 +1,41 @@
+package flak
+
+import (
+	. "fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+// Methods that many executors might use
+
+// Generates a temporary repeatr directory, creating all neccesary parent folders.
+// Must be passed at least one directory name, all of which will be used in the path.
+// Uses os.TempDir() to decide where to place.
+//
+// For example, GetTempDir("my-executor") -> /tmp/repeatr/my-executor/989443394
+func GetTempDir(dirs ...string) string {
+
+	if len(dirs) < 1 {
+		Println("Must have at least one sub-folder for tempdir", "replace with space monkey")
```

Comment:
other merges have gone through, you have the power.

i'd use `ProgrammerError` since this is a compile-time fuckup (`ValidationError` is the other potential sensible category, but those are more for when user input processing is involved and thus can make sense to catch in normal flow).